pkg/api/grpc/status.go

```diff
@@ -0,0 +1,54 @@
+package grpc
+
+import (
+	"context"
+
+	pb "github.com/stefanprodan/podinfo/pkg/api/grpc/status"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type StatusServer struct {
+	pb.UnimplementedStatusServiceServer
+	config *Config
+	logger *zap.Logger
+}
+
+// SayHello implements helloworld.GreeterServer
```

Comment:
Please remove this and the other lines below of commented code

Reply:
Did that! Removed unnecessary comments and whitespaces in code.
Please Check Once. Thanks